### ISSUE TITLE
Name of recipe is not visible in landscape mode.

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#5819]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yMAS; #A11yTCS; #Benchmark; #iOS; #iPad; #WCAG1.3.4; #GH_.NETMAUI_IOS_Mar2022; #Orientation​; #AINotInScope;

### Description

Name of recipe is not visible in landscape mode.

### Steps to Reproduce

1.Install and open ".NET MAUI" app.
2.Swipe till "My Recipes" and double tap.
3.Verify Name of recipe is not visible in landscape mode or Not.

https://user-images.githubusercontent.com/100186395/161698452-0e771dae-da09-4d80-8e39-d92e032499da.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15.4(iPad)

### Relevant log output

```shell
Name of recipe should be visible in landscape mode.
```

**Comment by Karan4495** [comment]
Verified issue in ".NET MAUI" app. issue is fixed so closing the bug
IOS Version: iPad(15.5)
added attachment for references

https://user-images.githubusercontent.com/100186395/177998911-2c25d377-f81d-49fe-a160-08214f907cf6.mp4
